docs/usage/configuration-options.md

```diff
@@ -2830,10 +2830,22 @@ It is recommended that you avoid using "negative" globs, like `**/!(package.json
 
 ### matchDepNames
 
+This field behaves the same as `matchPackageNames` except it matches against `depName` instead of `packageName`.
+
 ### matchDepPatterns
 
+<!-- prettier-ignore -->
+!!! note
+    `matchDepNames` now supports pattern matching and should be used instead.
+    Use of `matchDepPatterns` is now deprecated and will be migrated in future.
```

Comment:
we've a deprecation message property on config definition which should be use instead of manual admonition 😉

- https://github.com/renovatebot/renovate/pull/28216

Reply:
Won't that log a warning too?

Reply:
yes, to the dashboard 🫣